<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Video">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>790</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="videoLabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::StrongFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_Time">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_songTitle">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>19</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Volume">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_Volume">
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>626</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Previous">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Backward">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Play_Pause">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 105px;
    max-width: 110px;
    min-height: 110px;
    max-height: 110px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Forward">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Next">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenFile">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menuBluetooth">
    <property name="title">
     <string>Bluetooth</string>
    </property>
    <addaction name="actionAdd_Device"/>
   </widget>
   <addaction name="menuOpenFile"/>
   <addaction name="menuBluetooth"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionAdd_Device">
   <property name="text">
    <string>Add Device...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
